azure_data_cosmos_engine/src/query/mod.rs

```diff
@@ -87,6 +87,31 @@ impl PartitionKeyRange {
     }
 }
 
+#[derive(Clone, Debug, Deserialize)]
+#[cfg_attr(
+    feature = "python_conversions",
+    derive(pyo3::FromPyObject),
+    pyo3(from_item_all)
+)]
+#[serde(rename_all = "camelCase")]
```

Comment:
```suggestion
#[serde(rename_all = "PascalCase")]
```

And then you can maybe get rid of the renames below?

Reply:
The `Deserialize` implementation (which I assume is copied from one of our other models) is for Python compatibility (i.e. to "deserialize" an `ItemIdentity` from a Python object with `PartitionKeyValue` and `ID` properties).. We aren't prioritizing Python yet so I'd recommend we just get rid of it. We can always add it back in later, when we know more about how we'll structure the APIs. So you can just remove:

* The `Deserialize` from the `#[derive]` attribute
* The entire `#[cfg_attr]` attribute
* All `#[serde]` attributes from the type and fields.

Reply:
Oop! I see I'm wrong about that, it's used at the FFI level (Go passes a JSON-serialized value in here). If you're going to do that, I'd suggest keeping everything `camelCase` though to be consistent with other Cosmos DB backend APIs.